charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -80,14 +80,20 @@ data:
     {{- .Values.server.coordinatorExtraConfig | nindent 4 }}
     {{- end }}
 
-{{- if .Values.accessControl }}{{- if eq .Values.accessControl.type "configmap" }}
+{{- if .Values.accessControl }}
+  {{- if eq .Values.accessControl.type "configmap" }}
   access-control.properties: |
     access-control.name=file
     {{- if .Values.accessControl.refreshPeriod }}
     security.refresh-period={{ .Values.accessControl.refreshPeriod }}
     {{- end }}
     security.config-file={{ .Values.server.config.path }}/access-control/{{ .Values.accessControl.configFile | default "rules.json" }}
-{{- end }}{{- end }}
+  {{- end }}
+  {{- if eq .Values.accessControl.type "custom" }}
```

Comment:
Can you add a condition at the end to check if the type is set to anything other than `configmap` or `custom`, and call the `fail` template function with an actionable error message?

Ideally, we should also check if properties from different types are not mixed. For example, if `customValues` is _not_ set when `type` is `configmap`.

This kind of validation would require adding some more tests, but we don't have any negative tests in `test.sh` right now, so that's optional.